js/main.js

```diff
@@ -73,6 +79,22 @@ function getRecipe(event) {
   xhr.responseType = 'json';
   xhr.addEventListener('load', function () {
 
+    $recipeTitle.textContent = xhr.response.name;
+    $recipeImage.setAttribute('src', xhr.response.thumbnail_url);
+
+    for (var i = 0; i <= xhr.response.sections[0].components.length - 1; i++) {
+      var $li = document.createElement('li');
+      $li.textContent = xhr.response.sections[0].components[i].raw_text;
+      $recipeIngredients.appendChild($li);
+    }
+    for (var k = 0; k <= xhr.response.instructions.length - 1; k++) {
+      var $instructions = document.createElement('li');
+      $instructions.textContent = xhr.response.instructions[k].display_text;
+      $recipeInstructions.appendChild($instructions);
+    }
```

Comment:
If you look at a second recipe after the first one, does it have all the content from the previous recipe?

Reply:
![part-1](https://user-images.githubusercontent.com/93673754/148666597-981f7337-bbc0-449a-b075-aa5cde48c428.gif)
![part2](https://user-images.githubusercontent.com/93673754/148666599-d4a3b45d-36c5-4188-891c-cc211145fbba.gif)